python/fatcat_web/forms.py

```diff
@@ -472,7 +472,7 @@ def to_ingest_request(
         base_url = self.base_url.data
         ext_ids = release.ext_ids.to_dict()
         # by default this dict has a bunch of empty values
-        ext_ids = dict([(k, v) for (k, v) in ext_ids.items() if v])
```

Comment:
I also, subjectively, don't really like the curly-brace constructor syntax for dicts, though the explicit `dict()` around a list of tuples also isn't that great.

Reply:
On this one, we need to agree to disagree because I really find dict comprehensions easier to read and the `:` right up front makes it clear whether it is a dict or a set comprehension.